### ISSUE TITLE
[a11y support] 508-defect-2: Remaining Fixes for Form 40-10007 Apply for Pre-Need Eligibility (Out-of-Date with DST)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#62135]
## Point(s) of contact

 Josh Kim or Michelle Dieudonne Yenyo

## Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Remaining Items from Original Ticket
Details can be found in the next steps section.
- [ ] Password Field update
- [ ] Telephone Number in the Need Help sub-footer
- [ ] Content updates needed

## Reproduction Environments Available: Production or Stage
[This can be tested in Production or Staging](https://github.com/department-of-veterans-affairs/va.gov-team/issues/48297#issuecomment-1638011470), per @MichelleDieudonne 

See proposed solution for screens and previous ticket for further details (ticket #48297 )

- [Start the pre need form unsigned in](https://www.va.gov/burials-and-memorials/pre-need/form-10007-apply-for-eligibility/applicant-contact-information)
- Select "other" for relationship to service member
- Reach the upload page

### Password Field 
- The password field is required, if left blank, and not applied yet, and another file is uploaded, attempting to add a password to the first file does not appear to be working (in an unauthenticated test).  
- The password field will work on the second file. 
- Changing the second document's name appears to also cause an issue when typing the field highlight focus jumps to the PDF password field for the previous file. This makes it very difficult to type a new document name.
- Deleting the first file and uploading it again appears to rectify the problem; however, if the user does not apply a password, prior to uploading the next document, this is still an issue. 
![Password field issue.png](https://images.zenhubusercontent.com/63dbe59975ee0eef93e952b2/ba17f602-405a-4c8a-b934-699c80dc659b)
![Fields Working Correctly.png](https://images.zenhubusercontent.com/63dbe59975ee0eef93e952b2/61ab95f9-235a-4f40-a54b-a411454bf34c)

**Image Description:** Two images of file uploads, with the first image displaying the issue with typing into the second file name field, and the other showing the heading fields should not be bolded (strong) but should be semantic and follow the structure of the page. In this example: h3.

### Telephone Number
- The phone number in the "Need help?" section of the page in HTML is: `<a href="tel:18005351117">800-535-1117</a>`
- [According to the component for telephone numbers on the DST site](https://design.va.gov/components/telephone#content-considerations): `<a href="tel:+18005351117" aria-label="8 0 0. 5 3 5. 1 1 1 7.">800-827-1000</a>`

![DST vs Staging.png](https://images.zenhubusercontent.com/63dbe59975ee0eef93e952b2/02746ded-c818-4b7e-85e9-69d9a786270f)
**Image Description:** Comparison of the telephone number format on the DST site vs. the staging site, when inspecting the code. 

### Content Updates Needed

> [Content is out of date with the design system pattern](https://design.va.gov/patterns/ask-users-for/files)

It appears that the content is still out of date with the DST pattern. Most of the items needing fixed are below. Please review the full ["content considerations" ](https://design.va.gov/patterns/ask-users-for/files#content-considerations)section on the DST site for additional fixes.
- [ ] Mix of wording of documents and files. ... **Pattern recommends** using "**file**" instead of "document" 
    - [ ] 15 instances found on page.
- [ ] Form is using "add another" ... **Pattern recommends** "Upload" instead of "add."  
    - [ ] Change to: "add another file" 
- [X] "Delete file" is correct.
- [ ] Consider mobile users. ... **Pattern recommends** avoiding the word "scan" in file upload instructions. 

### References, articles, or WCAG support

@MichelleDieudonne noted in ticket #48297, @coforma-jamie and @laurwill can you take a look at the discrepancy between ["content considerations" ](https://design.va.gov/patterns/ask-users-for/files#content-considerations)section on the DST site for additional fixes, and what Michelle is referencing, below? 

> Regarding your content suggestions, we implemented changes to the review page content based on the examples in the https://design.va.gov/patterns/ask-users-for/files#examples page, which use words like "document" and "scan", so the guidance on that page overall is unfortunately contradictory.

### Type of issue

- [X] Screenreader
- [ ] Keyboard
- [X] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [X] Something else

**Comment by sara-amanda** [comment]
## CAIA Related Ticket

This is a new ticket for the spill over from #48297

**Comment by MichelleDieudonne** [comment]
@sara-amanda Are we able to request a severity change for this?  Since these changes would not block a release and would be expected to be completed by 1 or 2 sprints post release, we would give this a severity of 3, possibly a 4, since this would not be release blocking.

**Comment by coforma-jamie** [comment]
@MichelleDieudonne, we were able to figure out why there's a discrepancy between the content your team crafted and the [content considerations](https://design.va.gov/patterns/ask-users-for/files#content-considerations) - the [example images](https://design.va.gov/patterns/ask-users-for/files#examples) are out of date!

To clarify, please **use the content considerations**, NOT the example images as they are currently, as your source for truth for writing copy for this pattern.

There's a [ticket for the design system team](https://github.com/department-of-veterans-affairs/va.gov-team/issues/61470) that mentions that the example images for this pattern will be replaced.

Thank you for pointing this out, this is an issue we wouldn't have realized otherwise!

**Comment by ccruzgra** [comment]
@sara-amanda Thanks for creating this issue! We do have a suggestion for a correction to the AC for the below:

>  Form is using "add another" ... Pattern recommends "Upload" instead of "add."
 Change to: "add another file"

Based on the content considerations, this should be "Upload another file" and also in sentence case. See below for the guidance:

> Upload instead of add. Use the word “Upload” instead of “Add”. For example, “Upload file” and “Upload another file”.

Also worth noting that, as users come on the page, the button will first appear as "Upload file". Once they upload a file, the button label would then read as "Upload another file."

**Comment by MichelleDieudonne** [comment]
> @sara-amanda Are we able to request a severity change for this? Since these changes would not block a release and would be expected to be completed by 1 or 2 sprints post release, we would give this a severity of 3, possibly a 4, since this would not be release blocking.

 @sara-amanda @coforma-jamie Are either of you able to address the above?  Thanks!

**Comment by sara-amanda** [comment]
## CAIA Update
According to Platform's [severity rubric](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric), you are correct the content issues are not a severe issue. 

However, the identified severity level 2 aka "508-defect-2" now called "a11y-defect-2" still apply to the other remaining items, even though they are on a new ticket. They are post-launch issues, so you are good. We will just keep the label on there as is, as you work towards fixing those issues post release. 

@MichelleDieudonne @coforma-jamie @coforma-terry

**Comment by coforma-jamie** [comment]
Hi @MichelleDieudonne! Checking in on this ticket, how is it going?

**Comment by MichelleDieudonne** [comment]
This will be fixed by epic https://jira.devops.va.gov/browse/MBMS-44750

**Comment by MichelleDieudonne** [comment]
@coforma-jamie We had some issues with a part of this item.  Most of it will be fixed by EOD tomorrow and I will comment when you can review.  For the piece that's an issue, there is a platform thread for this in DSVA Slack. https://dsva.slack.com/archives/CBU0KDSB1/p1694528060923689

**Comment by coforma-jamie** [comment]
Thank you @MichelleDieudonne!

**Comment by coforma-jamie** [comment]
Just took a look at the staging site:

- **Phone number:** fixed!
- **Content updates:** done!
- **Password-protected updated:** fix in progress, as stated earlier :)

**Comment by coforma-jamie** [comment]
@MichelleDieudonne, I'm closing this ticket since the last issue will be resolved by Platform via [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/65382) and CAIA's work is complete. Thank you!
